components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -1170,20 +1174,14 @@ public static String getOAuth1AccessTokenUrl() {
 
         public static String getOAuth2AuthzEPUrl() {
```

Comment:
Other than meta data are there any other places where this method is invoked

Reply:
no